### DISCUSSION TITLE
Temporal Activities Hangfire

### DISCUSSION THREAD

**Discussion by marcprob** [elsa-workflows/elsa-core#1005] [Q&A] [answered]
Hello,

I wanted to test the hangfire temporal activities and checked out the current master branch and executed the Elsa.Sample.Timers.Hangfire project.
But it seems that the registered workflows, for example, CronTaskWorkflow,  are only executed once.
When I use the Quartz activities and change the cron definition it seems to be working fine.
Could there be a problem or do I have to configure something more?

Thanks for the answer.

**Comment by sfmskywalker** [comment, marked as answer]
I haven't tested the Hangfire implementation for a while, so it's definitely possible something got broken. I will investigate soon. Thanks for reporting!

**Reply by sfmskywalker** [reply]
I found a couple of issues that have been fixed with #1019 .

**Reply by marcprob** [reply]
Thanks a lot, now it works fine.
